cmd/kops/create_cluster.go

```diff
@@ -575,6 +580,18 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 		}
 	}
 
+	if c.MasterVolumeSize != "" {
+		for _, group := range masters {
```

Comment:
Do not think the default is "", for an int, we should default node and master to some size. I think we have an issue in for increasing the default. I do not believe it occurs in this code.